lib/index.js

```diff
@@ -18,6 +18,25 @@ module.exports.SyntaxError = Parser.SyntaxError;
 var astutil = require('./astutil');
 var contains_func = module.exports.contains_func = astutil.contains_func;
 
+function handleTexError(e, options) {
```

Comment:
Those two tasks are independent of each other, so you can go ahead and make the pull request. However, this is not high priority.